gta-gallery/src/layout/index.jsx

```diff
@@ -92,6 +68,42 @@ const Layout = () => {
     setVersions(versionsArray);
   };
 
+  const getBase64 = (img, callback) => {
```

Comment:
This method we have in 2 places. Please move to other file and use the same method all places